src/js/contentscript.js

```diff
@@ -168,13 +173,51 @@
             if (event.keyCode >= 32 && event.keyCode <= 126 && event.target.tagName) {
                 if (event.target.tagName.toUpperCase() === 'INPUT' ||
                         event.target.tagName.toUpperCase() === 'TEXTAREA' ||
-                        event.target.tagName.toUpperCase() === 'FORM') {
+                        event.target.tagName.toUpperCase() === 'FORM' ||
+                        event.target.isContentEditable === true) {
                     inputState = true;
                     var tabState = buildTabStateObject();
                     chrome.runtime.sendMessage(tabState);
+                    //stores so it can be checked later for unsaved values
+                    editedInputControls.push(event.target);
+                }
+            }
+        }
+    }
+    
+    function formInputChecker() {
+        var newEditedInputControls = [];
+        for (var el of editedInputControls) {
+            if (isElementInBody(el) === true) {
```

Comment:
Why do you check if the element in in a body tag? Does it matter?

Reply:
In case it gets dynamically removed. If a script removes it, I believe the reference will still prevent the garbage disposal from setting it to null, so could lead to phantom elements that are being checked for edits, but don't actually exist in the DOM.